libtrap/src/ifc_file.c

```diff
@@ -428,9 +460,9 @@ int create_file_recv_ifc(trap_ctx_priv_t *ctx, const char *params, trap_input_if
    priv->ifc_idx = idx;
    /* Perform shell-like expansion of ~ */
    if (wordexp(params, &files_exp, 0) != 0) {
-      VERBOSE(CL_ERROR, "CREATE INPUT FILE IFC: unable to perform shell-like expand of: %s", params);
+      VERBOSE(CL_ERROR, "FILE INPUT IFC[%u]: Unable to perform shell-like expansion of: %s", idx, params);
       free(priv);
-      return trap_errorf(ctx, TRAP_E_BADPARAMS, "CREATE INPUT FILE IFC: unable to perform shell-like expand");
+      return trap_errorf(ctx, TRAP_E_BADPARAMS, "FILE INPUT IFC[%u]: Unable to perform shell-like expansion.", idx);
```

Comment:
General note for this line and above lines: I'd prefer PRIuXX according to data type length...